geopy/distance.py

```diff
@@ -120,6 +120,31 @@
     'GRS-67':        (6378.1600, 6356.774719, 1 / 298.25)
 }
 
+
+def lonlat(x, y, z=0):
+    """
+    `geopy.distance.distance` accepts coordinates in ``(y, x)/(lat, lon)`` order, while some other libraries/systems
+    might use ``(x, y)/(lon, lat)``.
+
+    This function provides a convenient way to convert coordinates of the ``(x, y)/(lon, lat)`` format to
+    a :class:`geopy.point.Point` instance.
+
+    Example::
+    
+    >>> from geopy.distance import lonlat, distance
```

Comment:
Code blocks should be indented.

Like this: https://github.com/svalee/geopy/blob/f4d0b509dccc1f8a937e1ae2416066bcf611f08b/geopy/distance.py#L289